src/main/scala/is/hail/methods/LDPrune.scala

```diff
@@ -2,15 +2,68 @@ package is.hail.methods
 
 import java.util
 
+import is.hail.HailContext
 import is.hail.annotations._
 import is.hail.expr.types._
+import is.hail.linalg.BlockMatrix
 import is.hail.sparkextras.GeneralRDD
 import org.apache.spark.storage.StorageLevel
-import is.hail.sparkextras._
 import is.hail.rvd.{OrderedRVD, OrderedRVDType, RVD, UnpartitionedRVD}
+import is.hail.table.Table
 import is.hail.variant._
 import is.hail.utils._
-import org.apache.spark.sql.Row
+
+import scala.collection.mutable
+
+object LDPruneSparseEntriesTable {
+  def windows(tbl: Table, window: Int): List[(Long, Long)] = {
+    val expectedSignature = TStruct("index" -> TInt64(), "chromosome" -> TString(), "position" -> TInt32())
```

Comment:
I think whatever method is producing the table should use the more general term "contig" (meaning a contiguous region of DNA) rather than "chromosome". That's what we define on Locus:
https://hail.is/docs/devel/genetics/hail.genetics.Locus.html?highlight=locus

If this piece is a bottleneck, it can be parallelized by `contig`, even locally, using parallel collections.

Reply:
I broke this method into parts and renamed it to computeIntervalsOfNearbyLociByIndex. I didn't look at parallelizing by contig. It takes about 7 seconds to run on a table with 12,000 variants.